.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -113,6 +119,8 @@ jobs:
             encryption: 'ipsec'
             encryption-node: 'false'
             endpoint-routes: 'true'
+            key-one: 'gcm(aes)'
+            key-two: 'gcm(aes)'
```

Comment:
Maybe worth testing the reverse, cbc -> gcm?

Reply:
Good point, let me add it!